src/unix/newlib/mod.rs

```diff
@@ -0,0 +1,566 @@
+use dox::mem;
```

Comment:
Or not. Turns out I actually use that import after all. `FD_CLR()` and friends depend on it and I'd just forgotten to include them last time around.